tests/backup/backup_scale_test.go

```diff
@@ -0,0 +1,212 @@
+package tests
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/pborman/uuid"
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+)
+
+// MutipleBackupLocationWithSameEndpoint Create Backup and Restore for Mutiple backup location added using same endpoint.
+var _ = Describe("{MutipleBackupLocationWithSameEndpoint}", func() {
+	var (
+		scheduledAppContexts          []*scheduler.Context
+		backupLocationNameMap         = make(map[int]string)
+		backupLocationUIDMap          = make(map[int]string)
+		backupLocationMap             = make(map[string]string)
+		bkpNamespaces                 []string
+		cloudCredName                 string
+		cloudCredUID                  string
+		clusterUid                    string
+		labelSelectors                map[string]string
+		mutex                         sync.Mutex
+		wg                            sync.WaitGroup
+		userBackupMap                 = make(map[int]map[string]string)
+		restoreNames                  []string
+		numberOfBackupLocation        = 1000
+		numberOfBackups               = 30
+		providers                     = getProviders()
+		timeBetweenConsecutiveBackups = 10 * time.Second
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("MutipleBackupLocationWithSameEndpoint", "Create Backup and Restore for Mutiple backup location added using same endpoint", nil, 86088)
+		log.InfoD("scheduling applications")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts := ScheduleApplications(taskName)
+			for _, appCtx := range appContexts {
+				appCtx.ReadinessTimeout = appReadinessTimeout
+				scheduledAppContexts = append(scheduledAppContexts, appCtx)
+				namespace := GetAppNamespace(appCtx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+
+	It("Create Backup and Restore for Mutiple backup location added using same endpoint", func() {
+		Step("Validate applications", func() {
+			log.InfoD("Validating applications")
```

Comment:
Can we move this below ValidateApplications?

Reply:
Done